### PR TITLE
Pseudoqueue

### PR DESCRIPTION
**Pull request by shaybrow**
- [x] Top-level README “Table of Contents” is updated
 - [x] Feature tasks for this challenge are completed
 - [x] Unit tests written and passing
     - [x] “Happy Path” - Expected outcome
     - [x] Expected failure
     - [ ] Edge Case (if applicable/obvious)
 - [x] README for this challenge is complete
     - [x] Summary, Description, Approach & Efficiency, Solution
     - [x] Link to code
     - [x] Picture of whiteboard

### CODE REVIEW

File: challenges/lib/src/main/java/challenges/stackandqueues/PseudoQueue.java [hunk 1]
```diff
@@ -0,0 +1,25 @@
+package challenges.stackandqueues;
+
+public class PseudoQueue {
```
Comment (Mattpet26) [inline challenges/lib/src/main/java/challenges/stackandqueues/PseudoQueue.java:3]: I'm confused why you chose to name this front and back? These are two separate stacks.

File: challenges/lib/src/main/java/challenges/stackandqueues/PseudoQueue.java [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    public PseudoQueue() {
+        this.front = new Stack();
+        this.back = new Stack();
+    }
+    public void enqueue (int input){
+
+        back.push(input);
+    }
+    public int dequeue () throws Exception {
+        if (this.front.isEmpty() == true){
```
Comment (Mattpet26) [inline challenges/lib/src/main/java/challenges/stackandqueues/PseudoQueue.java:16]: within the if statement, you don't need to set isEmpty() to == true. The method already returns a boolean. If you want to check for falsey, just do !isEmpty() (is not empty)
